scripts/prepare_base_image_buildinfo.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+
+ARCH=$1
+DISTRO=$2
+FILESYSTEM_ROOT=$3
+
+. /usr/local/share/buildinfo/scripts/buildinfo_base.sh
+VERSION_DEB_PREFERENCE="01-versions-deb"
+BUILDINFO_PATH=${FILESYSTEM_ROOT}/usr/local/share/buildinfo
+BUILDINFO_VERSION_PATH=${FILESYSTEM_ROOT}/usr/local/share/buildinfo/versions
+BUILDINFO_VERSION_DEB=${BUILDINFO_VERSION_PATH}/${VERSION_DEB_PREFERENCE}
+OVERRIDE_VERSION_PATH=files/build/versions/host-image
+DIFF_VERSIONS_PATH=$BUILDINFO_PATH/diff-versions
+
+mkdir -p $BUILDINFO_PATH
+
+# Copy the build info config
+cp -rf files/build/buildinfo/* $BUILDINFO_PATH/
+
+# Install the config files
+cp $BUILDINFO_PATH/trusted.gpg.d/* "${FILESYSTEM_ROOT}/etc/apt/trusted.gpg.d/"
```

Comment:
Move to the section "## Pointing apt to public apt mirrors and getting latest packages, needed for latest security updates" inside build_debian.sh